### PR TITLE
feat: do not use podIP as the address of the node

### PR DESCRIPTION
**Pull request by OrangeBao**
#### What type of PR is this?

/kind feature
#### Which issue(s) does this PR fix?

Fixes #
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/clustertree/cluster-manager/utils/leaf_model_handler.go [hunk 1]
```diff
@@ -177,7 +194,7 @@ func (h DispersionModelHandler) CreateNodeInRoot(ctx context.Context, cluster *k
 					},
 				}
 
-				node.Status.Addresses = GetAddress()
+				// node.Status.Addresses = GetAddress()
```
Comment (hanweisen) [inline pkg/clustertree/cluster-manager/utils/leaf_model_handler.go:197]: 注释代码删除

File: pkg/clustertree/cluster-manager/utils/rootcluster.go [hunk 2]
```diff
@@ -22,9 +27,64 @@ func IsRootCluster(cluster *kosmosv1alpha1.Cluster) bool {
 	return false
 }
 
-func GetAddress() []corev1.NodeAddress {
-	address := []corev1.NodeAddress{
-		{Type: corev1.NodeInternalIP, Address: os.Getenv("KNODE_POD_IP")},
+func GetAddress(ctx context.Context, rootClient kubernetes.Interface, originAddress []corev1.NodeAddress) ([]corev1.NodeAddress, error) {
+	// address := []corev1.NodeAddress{
```
Comment (hanweisen) [inline pkg/clustertree/cluster-manager/utils/rootcluster.go:31]: 注释代码删除

File: pkg/clustertree/cluster-manager/controllers/node_resources_controller.go [hunk 3]
```diff
@@ -143,13 +143,17 @@ func (c *NodeResourcesController) Reconcile(ctx context.Context, request reconci
...
-					Taints: rootNode.Spec.Taints,
-				}
-
-				clone.Spec = spec
+				// TODO @duanmengkk
+				// spec := corev1.NodeSpec{
+				// 	Taints: rootNode.Spec.Taints,
+				// }
+				clone.Spec.Taints = rootNode.Spec.Taints
 				clone.Status = node.Status
```
Comment (OrangeBao) [inline pkg/clustertree/cluster-manager/controllers/node_resources_controller.go:151]: /cc @duanmengkk

**Pull request merged**